github/actions_cache.go

```diff
@@ -0,0 +1,236 @@
+// Copyright 2022 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ActionsCache represents a GitHub action cache.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/cache?apiVersion=2022-11-28#about-the-cache-api
+type ActionsCache struct {
+	ID             *int64     `json:"id,omitempty" url:"-"`
+	Ref            *string    `json:"ref,omitempty" url:"ref"`
+	Key            *string    `json:"key,omitempty" url:"key"`
+	Version        *string    `json:"version,omitempty" url:"-"`
+	LastAccessedAt *Timestamp `json:"last_accessed_at,omitempty" url:"-"`
+	CreatedAt      *Timestamp `json:"created_at,omitempty" url:"-"`
+	SizeInBytes    *int64     `json:"size_in_bytes,omitempty" url:"-"`
+}
+
+// ActionsCacheList represents a list of GitHub actions Cache.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/cache?apiVersion=2022-11-28#list-github-actions-caches-for-a-repository
+type ActionsCacheList struct {
+	TotalCount    *int64          `json:"total_count,omitempty"`
```

Comment:
I think this one is safe to leave as `*int` (not `*int64` as we tend to reserve that for really large numbers or numeric IDs).